Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnSelect.hs

```diff
@@ -106,7 +106,7 @@ mkQuoteEntities dReq quote = do
 mkFulfillment :: DOnSelectReq -> DQuote.DriverQuote -> OS.FulfillmentInfo
 mkFulfillment dReq quote = do
   let fromLocation = dReq.searchRequest.fromLocation
-  let toLocation = dReq.searchRequest.toLocation
+  let toLocation = last dReq.searchRequest.toLocation
```

Comment:
@roman-bodavskiy can you help here I am not able to use this:
`  destination <- lastMaybe mbToLocation & fromMaybeM (InternalError "To location not found.")`